src/h5web/toolbar/Toolbar.tsx

```diff
@@ -24,48 +24,41 @@ function Toolbar(props: Props) {
   const childrenWidths = useMap<string, number>();
 
   // Group visible and hidden children based on their accumulated width
-  const [visibleChildren, hiddenChildren, allMeasured] = allChildren.reduce<
-    [ReactElement[], ReactElement[], boolean, number]
+  const [visibleChildren, hiddenChildren] = allChildren.reduce<
```

Comment:
Rereading this, I feel the naming `visible/hiddenChildren` can be misleading.

What about `displayed` instead of `visible` and `overflowing` instead of `hidden` ?

Reply:
I went with `inView` and `outOfView` to keep the names short and avoid excessive line wrapping.